src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -116,6 +122,32 @@ public void itemStateChanged(ItemEvent e) {
 
         toolbar.add(Box.createHorizontalStrut(16));
         toolbar.add(showErrorsToggleButton);
+        
+        /**
+         * Represents the 'Export' button	
+         */
+        toolbar.add(Box.createHorizontalStrut(32));
+        JPanel buttonPanel = new JPanel();   
+        buttonPanel.add(new JButton(new AbstractAction("Export") {
+			private static final long serialVersionUID = 1L;
+
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				JFileChooser chooser = new JFileChooser();
+				if (chooser.showSaveDialog(null) == JFileChooser.APPROVE_OPTION) {
+					try (PrintWriter pw = new PrintWriter(chooser.getSelectedFile())) {
+		        		int count = currentFuzzer.getMessagesModel().getRowCount();
+		        		for (int i = 0; i < count; i++) {

```

Comment:
getRowCount() returns the number of rows from an ArrayList. Since I iterate through a List, I start with 0 since elements are indexed as n - 1. I tested it and it includes all the rows :)